endpoints/2-cloud-endpoints.md

```diff
@@ -134,7 +134,8 @@ Contains SSH access information for the cloud. For Open Stack and AWS should be
 
     - `username` (string): The SSH username.
     - `keyPairName` (string): The name of the key pair used for SSH access.
-    - `privateKey` (string or `null`): The private key in RSA format, with line breaks represented by `\n` for JSON compatibility. If not required, use `null`.
+    - `publicKey` (string or `null`): The public key in RSA format. If not required, use `null`.
```

Comment:
do we have publicKey for add cloud. i think it is just private one.

Reply:
Thx to this PR .. yes!